website/README.md

```diff
@@ -45,22 +45,28 @@ You should preview all of your changes locally before creating a pull request. T
 
 
 ## Deploying Changes
-The website generates versioned documentation by pointing to the HEAD of the release branch for that version. For example, the `v1.2.x` documentation on the website points to the HEAD of the `v1.2` release branch in the `terraform` repository. 
 
-Merging a PR to `main` queues up documentation changes for the next minor product release. To update the latest documentation, you must also backport your changes to one or more release branches. Changes that you push to a release branch become live on the site within one hour.
+Merging a PR to `main` queues up documentation changes for the next minor product release. Your changes are not immediately available on the website.
 
-For example, if Terraform is on v1.2.x:
-- Merge to `main`: Documentation for v1.3 features.
-- Merge to `main` and backport to `v1.2` release branch: Typo fixes, page restructures, documentation to support v1.2 patch releases, etc.
+The website generates versioned documentation by pointing to the HEAD of the release branch for that version. For example, the `v1.2.x` documentation on the website points to the HEAD of the `v1.2` release branch in the `terraform` repository. To update existing documentation versions, you must also backport your changes to that release branch. Backported changes become live on the site within one hour.
 
 ### Backporting
 
 **Important:** Editing old versions (not latest) should be rare. We backport to old versions when there is an egregious error. Egregious errors include inaccuracies that could cause security vulnerabilities or extreme inconvenience for users. 
 
-Backporting involves cherry-picking commits to one or more release branches within a docs repository. For example, if Terraform is currently on v1.2 and you need to add a security warning to the v1.1 documentation, you can backport (cherry-pick) commits to the v1.1 branch by labeling the PR with a backport label (e.g., 1.2-backport) associated with the release branch for the target version. 
+Backporting involves cherry-picking commits to one or more release branches within a docs repository. You can backport (cherry-pick) commits to a version branch by adding the associated backport label to your pull request. For example, if you need to add a security warning to the v1.1 documentation, you must add the `1.1-backport` label. When you merge a pull request with one or more backport labels, GitHub Actions opens a backport PR to cherry-pick your changes to the associated release branches. Someone needs to manually merge the backport PR to finish backporting the changes. 
```

Comment:
Suggest changing "Someone" tin the final sentence o be a bit more specific as to who is able to manually merge the backport. Can literally anyone from anywhere do it? A member of the user's team? Or whom?

Reply:
fixed!! thank you!